stages/org.osbuild.ostree.encapsulate

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+"""
+Wrap OSTree commits into a oci container image
```

Comment:
Awesome, thanks so much for starting this! :+1: 

It may make sense to expose this as is...but for osbuild I think we should be focusing on `rpm-ostree compose container-encapsulate` - more on the rationale for that over at https://coreos.github.io/rpm-ostree/container/#converting-ostree-commits-to-new-base-images

This is how the current Fedora CoreOS images are made, and also what is used to implement https://pagure.io/releng/issue/11047

(The underlying Rust APIs in ostree-ext do support generating "chunked" images, and I could imagine that osbuild could grow support for doing this directly...but the logic for "chunking" is quite important here, and we're actively working on improving the rpm logic here in rpm-ostree)

Reply:
(Oh wow I completely somehow missed that you're already doing this, ignore the comment above)